app/controllers/dreams_controller.rb

```diff
@@ -24,9 +24,25 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    @dream.update(dream_params)
+  end
+
+  def destroy
+    @dream.destroy
+    redirect_to root_path status: :see_other
```

Comment:
I think the comma is missing after rooth_path, but maybe not impacting the code

   "redirect_to root_path, status: :see_other"